src/components/EventCard.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import { CardContent, Typography } from '@material-ui/core';
+import CardActions from '@material-ui/core/CardActions';
+import Button from '@material-ui/core/Button';
+import useStyles from './util/useStyles';
+const EventCard = ({ title, description, startDate, endDate }) => {
+	const classes = useStyles();
+	return (
+		<Card className={classes.eventCard}>
+			<CardContent>
+				<Typography variant={'h3'}>
+					{title}
+				</Typography>
+				<Typography variant={'h5'} color="textSecondary">
+					{description.substring(0, 50)}
```

Comment:
Could you try using the `noWrap` property documented [here](https://material-ui.com/api/typography/) instead of a manual substring?